pkgs/development/libraries/libcacard/default.nix

```diff
@@ -0,0 +1,14 @@
+{ stdenv, fetchurl, pkgconfig, glib, nss }:
+
+stdenv.mkDerivation rec {
+  name = "libcacard-${version}";
+  version = "2.6.1";
+
+  src = fetchurl {
+    url = "https://www.spice-space.org/download/libcacard/${name}.tar.xz";
+    sha256 = "1w6y0kiakhg7dgyf8yqpm4jj6jiv17zhy9lp3d7z32q1pniccxk2";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ glib nss ];
+}
```

Comment:
Since this is exposed in `all-packages.nix` it should have the usual `meta` info.
As an alternative, you could make it local to `spice` if it's unlikely to have other uses.

Reply:
Oh, right, sorry.